steely/plugins/btc.py

```diff
@@ -1,33 +1,64 @@
 '''Get the latest Bitcoin price (in USD and EUR)'''
 import requests
 
-__author__ = 'CianLR'
+__author__ = ('CianLR', 'byxor')
 COMMAND = 'btc'
 
 API_URL = 'https://api.coindesk.com/v1/bpi/currentprice.json'
 RESP_TEMPLATE = """Bitcoin Price:
-    €{}
-    ${}"""
+€{}
+${}
+Difference since last time: {}"""
```

Comment:
i think just make this a blank {}% thing

Reply:
That responsibility is already satisfied by `percentage_string`. Why move it and break tests?

Reply:
i mean like remove "difference since ...."  I like that the output is simple

Reply:
I think "Difference since last time: {}" might even go onto 2 lines on mobile. Just say "Delta: {}%" or "Diff: {}%" would be better :+1: